src/coreclr/tests/testenvironment.proj

```diff
@@ -138,6 +138,8 @@
     <TestEnvironment Include="jitguardeddevirtualization" JitEnableGuardedDevirtualization="1" TieredCompilation="0" />
     <TestEnvironment Include="jitehwritethru" EnableEhWriteThru="1" TieredCompilation="0" />
     <TestEnvironment Include="jitobjectstackallocation" JitObjectStackAllocation="1" TieredCompilation="0" />
+    <!-- Mono specific options -->
+    <TestEnvironment Include="interpreter" MONO_ENV_OPTIONS="--interpreter"/>
```

Comment:
@naricc Is my understanding correct that you are trying to set `MONO_ENV_OPTIONS` environment variable to `--interpreter` value here?

If so, then this is not going to work - since setting TestEnvironment metadata `M` to some value `V` here will effectively do `set COMPlus_M=V` but not `set M=V`.

Reply:
Hmm, I see. What *is* the correct way to do what I want? Set MONO_ENV_OPTIONS=="--interpreter"?

Reply:
maybe we could add an optional metadata to `TestEnvironment`, something like `IgnoreComPlus=false` -- so whoever expands these environment variables, checks if that metadata value is == false, if it is, it just sets the env var as the name of the metadata.

Reply:
If setting this environment variable is all you need then I would add something like 
```xml
  <_TestEnvFileLine Include="set MONO_ENV_OPTIONS=--interpreter" Condition="'$(Scenario)' == 'interpreter'" />
```
and similar for Unix
```xml
  <_TestEnvFileLine Include="export MONO_ENV_OPTIONS=--interpreter" Condition="'$(Scenario)' == 'interpreter'" />
```
to CreateTestEnvFile target down below. 

If in the future, there will be more cases like this - then we can extend this project and add another target  "OutputRawVariables" - that will compare a TestEnvironment metadata name against the list of pre-defined environment variables names - the same way it's done in OutputCOMPlusVariables. But I don't think this is needed for that particular case.

BTW, you can test your changes with this command
```
dotnet msbuild testenvironment.proj /p:TestEnvFileName=SetStressModes.cmd /p:Scenario=interpreter /p:TargetsWindows=true
```
and check content of SetStressModes.cmd file